.install/ubuntu_22.04.sh

```diff
@@ -4,5 +4,8 @@ export DEBIAN_FRONTEND=noninteractive
 MODE=$1 # whether to install using sudo or not
 
 $MODE apt-get update -qq || true
-$MODE apt-get install -yqq git wget build-essential valgrind lcov
+$MODE apt-get install -yqq gcc-12 g++-12 git wget build-essential valgrind lcov
```

Comment:
why not 13?

Reply:
GCC13 requires ppa and i wanted the keep the installation as simple as possible